### ISSUE TITLE
RFP - GoPro Quik

### ISSUE THREAD

**Issue by aster94** [chocolatey-community/chocolatey-package-requests#541]
## Checklist

- [x] The package I am requesting does not already exist on https://chocolatey.org/packages;
- [x] There is no open issue for this package;
- [x] The issue title starts 'RFP - ';
- [x] The download URL is public and not locked behind a paywall / login;

## Package Details

Software project URL : https://shop.gopro.com/softwareandapp/quik-%7C-desktop/Quik-Desktop.html#
Direct download URL for the software / installer : i can't copy the url but it is in the page above
Software summary / short description:
Import and enjoy your GoPro footage and create beautiful videos with just a few clicks.

•	Easily transform your footage into awesome videos synced to music.
•	Automatically import1 your photos and videos and share your best shots.
•	Add gauges and graphs that show how fast, far and high you went.2

**Comment by chtof** [comment]
I haven't find option to launch the installer in silent mode (https://software.gopro.com/PC/GoPro_Quik-WinInstaller-2.7.0.945.exe).
So, I have tried with AutoHotkey: Win Spy doesn't identify window controls like buttons (only the window title is identified) so I have tried to click on button using coordinates: it doesn't work.
If someone want to have a look (an AutoHotkey Guru ;-) ), the package is available here: https://github.com/chtof/chocolatey-packages/tree/master/automatic/quik
Thanks in advance for your help.
